linux-tools/acl/acl.sh

```diff
@@ -79,7 +79,10 @@ function runtest()
     #$TESTDIR/run $cmd >$stdout 2>$stderr
     cd $TESTDIR
     ./run $cmd >$stdout 2>$stderr
-    tc_fail_if_bad $? "unexpected response" || return
+    RC=$?
+        [ $RC -eq 0 ] && tc_ignore_warnings "Unescaped left brace in regex is deprecated, passed through"
+
```

Comment:
It appears to me that this error comes from perl ( version 5.22 ) and this is fixed in later version of perl. 
And it can be fixed by escaping '{'.  Can you please verify if this needs to ignored only for this version of perl ?

Please modify summary to be more precise on the changes.
For each of the changes, better to add in the comments on why it is happening in ubuntu environment specifically. like package/environment changes ?